scripts/basic-workflow-tests.sh

```diff
@@ -52,8 +52,13 @@ test_init() {
     task Initializing ilab
     [ -f config.yaml ] || ilab config init --non-interactive
 
-    step Checking config.yaml
-    grep merlinite config.yaml
+    step Checking model configuration values
```

Comment:
I believe all these tests should be added to `scripts/functional-tests.sh` instead. This file is used for end-to-end testing.

Reply:
That seems like a good idea. What do you think @russellb?

Reply:
@hickeyma thanks for considering it and happy to see more tests :D

Reply:
yes, agree with the move

Reply:
Ok, updated 59449375652641afde4614a6ee19a2d3823ce816 to move tests to functional testing and using `yk` instead of `grep`.